rollup.config.js

```diff
@@ -48,6 +48,20 @@ function strings({ include, exclude } = {}) {
   };
 }
 
+// This plugin resolves at bundle time the `msg_generated` module
+function resolveMsgGenerated() {
+  return {
+    name: "resolve-msg-generated",
+    resolveId(importee) {
+      if (importee === "msg_generated") {
+        return path.resolve(
+          path.join(process.cwd(), "gen", "msg_generated.ts")
```

Comment:
Valid point.  This _may_ not be the only generated asset that ends up in the bundle, so I should change it to be a more generic too.  So that any MID that starts with `gen/*` gets remapped to the current build directory.